src/Core/DockerContainerManager.cs

```diff
@@ -316,7 +317,7 @@ private async Task CreateContainerAsync()
                 portMapping.Value.Add(
                     new PortBinding()
                     {
-                        HostIP = "",
+                        HostIP = containerPortMapping.AllowedHostIp ?? "",
```

Comment:
I guess `AllowedHostIp` should be called `HostIp`

Reply:
But I will change it.